nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -306,7 +298,9 @@ class TfOpIdxMap {
 
     auto &buffer_map = getIndexMap<const float *, Buffer>();
     buffer_map.addDataWhenNotFound(
-      empty_buffer, {0, empty_buffer}); /// put empty buffer to first
+      nullptr, {0, empty_buffer}); // this represents undefined buffer
```

Comment:
when would this be needed?

Reply:
This is part of tflite file spec. the first buffer index denotes undefined buffer.

Reply:
got it.